extensions/interactions/Continue/directives/oppia-response-continue.component.spec.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the Continue button response.
+ */
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { OppiaResponseContinueComponent } from './oppia-response-continue.component';
+import { HtmlEscaperService } from 'services/html-escaper.service';
+
+describe('OppiaResponseContinueComponent', () => {
+  let component: OppiaResponseContinueComponent;
+  let fixture: ComponentFixture<OppiaResponseContinueComponent>;
+
+  class mockHtmlEscaperService {
+    escapedJsonToObj(answer): string {
+      return answer;
+    }
+  }
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [OppiaResponseContinueComponent],
+      providers: [
+        {
+          provide: HtmlEscaperService,
+          useClass: mockHtmlEscaperService
+        }
+      ]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(OppiaResponseContinueComponent);
+    component = fixture.componentInstance;
+  });
+
+  it('should create', () => {
```

Comment:
it should create what? The description here can be changed to explain what you are testing.

Reply:
Done